de/manual/ch_GUIMenus.xml

```diff
@@ -6665,459 +6665,1082 @@ url="&url-docs-de;guide/chapter_txns.html#txns-regstyle1"> &app; Kurs und Konzep
-      <title>Kundenübersicht</title>
+      <para>Dieser Abschnitt beschreibt die Fensterelemente für die Anwendungsfälle
+        <emphasis>Rechnungsbearbeitung</emphasis> und <emphasis>Kundenübersicht</emphasis>.
+      </para>
 
-      <para>Um die Anzeige der <guilabel>Kundenübersicht</guilabel> zu öffnen, klicken Sie auf
-        &mc.bus.cus.o-view; .
+      <para>Die Funktionsweise der Dialoge und Bedienelemente für die Geschäftsfunktionen
+        werden ausführlich im Kapitel <xref linkend="chapter_busnss" /> beschrieben.
       </para>
+    </abstract>
 
-      <table pgwide="1" id="business-customer-overview-toolbar-table">
-        <title>Kunden - Werkzeugleiste</title>
+    <note>
+      <para>Da sich die Fensteransichten und die dazugehörigen Menüs für <emphasis>Kunden</emphasis>, 
+        <emphasis>Lieferanten</emphasis> und <emphasis>Mitarbeiter</emphasis> nicht unterscheiden,
+        werden diese hier nur einmal beschrieben.
+      </para>
+    </note>
 
-        <tgroup cols="2">
-          <thead>
-            <row>
-              <entry>
-                <para>Schaltfläche
-                </para>
-              </entry>
+    <sect2 id="Gui-business-invoice">
+      <title>Fensteransicht für Rechnungsfunktionen</title>
+      
+      <figure pgwide="1">
+        <title>Rechnung bearbeiten</title><screenshot>
+        <screeninfo>1431x776x256</screeninfo>
+          <mediaobject>
+            <imageobject role="html">
+              <imagedata fileref="figures/GUI_Business_Invoice.png" srccredit="Christian Wehling" width="&img-w;" />
+            </imageobject>
 
-              <entry align="center">
-                <para>Beschreibung
-                </para>
-              </entry>
-            </row>
-          </thead>
+            <imageobject role="fo">
+              <imagedata fileref="figures/GUI_Business_Invoice.png" srccredit="Christian Wehling" />
+            </imageobject>
 
-          <tbody>
-            <row>
-              <entry>
-                <para><guibutton>Neu</guibutton>
-                </para>
-              </entry>
+            <caption>
+              <para>Die Ansicht zur Bearbeitung von <emphasis>Rechnungen</emphasis> und
+                    <emphasis>Lieferantenrechnungen</emphasis>.
+              </para>
+            </caption>
+          </mediaobject>
+        </screenshot>
+      </figure>
 
-              <entry>
-                <para>Den Dialog zum Anlegen eines neuen Kunden öffnen.
-                </para>
-              </entry>
-            </row>
+    <sect3 id="Gui-business-invoice-display">
+      <title>Anzeigebereich</title>
 
-            <row>
-              <entry>
-                <para><guibutton>Bearbeiten</guibutton>
-                </para>
-              </entry>
+      <para>In dieser Ansicht können Sie sowohl <emphasis>(Kunden-)Rechnungen</emphasis> und
+        <emphasis>Lieferantenrechnungen</emphasis> als auch <emphasis>Auslagenerstattungen</emphasis>
```

Comment:
Ein Nachteil der Auslagen-Ersttattung ist, daß die Steuer-Spalten fehlen.
Für die deutsche implementierung der MWSt. als USt. mit Vorsteuerabzug ist das nicht so schön.

Aber bei meinen Importen habe ich die Zoll-Abrechnungen auch in dem Rechnungsformular abgbildet:
Präferenzware [zollfrei] (also am Jahresanfang importieren, wenn noch Quoten frei sind)
Warengruppe 1
Zoll x% auf WG 1
Warengruppe 2
Zoll y% auf WG 2
…
Transportkosten außerhalb der EU
Einfuhr-Umsatzsteeuer (=MWSt) auf die Zwischensumme
Transportkosten innerhalb der EU
…